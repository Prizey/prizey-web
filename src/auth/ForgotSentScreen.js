import React from 'react'
import Layout from 'design/Layout/Layout'
import { Typography } from '@material-ui/core'

export default () => (
  <Layout>
    <Typography variant="h6" align="center">
      Instructions for reseting your password have been sent to your email
    </Typography>
  </Layout>
)
