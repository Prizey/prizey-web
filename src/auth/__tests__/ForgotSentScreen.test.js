import React from 'react'
import renderer from 'react-test-renderer'
import ForgotSentScreen from '../ForgotSentScreen'

jest.mock('../../design/Layout/RegisterPageView')

it('renders correctly', () => {
  const tree = renderer.create(<ForgotSentScreen />).toJSON()
  expect(tree).toMatchSnapshot()
})
