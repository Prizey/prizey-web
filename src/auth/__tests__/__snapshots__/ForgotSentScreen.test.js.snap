// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Component-root-8"
>
  <div
    className="Component-header-6"
  />
  <div
    className="Component-container-3"
  >
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      <img
        alt="Prizey"
        className="Component-logoImage-7"
        src="/logo.png"
      />
    </a>
    <div
      className="Component-caption-1"
    />
    <div
      className="Component-children-2"
    >
      <h6
        className="MuiTypography-root-9 MuiTypography-h6-26 MuiTypography-alignCenter-32"
      >
        Instructions for reseting your password have been sent to your email
      </h6>
    </div>
    <div
      className="Component-footer-4"
    >
      <a
        className="Component-footerLink-5"
        href="/terms_of_service"
        onClick={[Function]}
        rel="noopener noreferer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-9 MuiTypography-caption-19 MuiTypography-colorTextSecondary-42 MuiTypography-alignCenter-32"
        >
          TERMS
        </span>
      </a>
      <a
        className="Component-footerLink-5"
        href="/privacy-policy"
        onClick={[Function]}
        rel="noopener noreferer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-9 MuiTypography-caption-19 MuiTypography-colorTextSecondary-42 MuiTypography-alignCenter-32"
        >
          PRIVACY
        </span>
      </a>
      <a
        className="Component-footerLink-5"
        href="/faq"
        onClick={[Function]}
        rel="noopener noreferer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-9 MuiTypography-caption-19 MuiTypography-colorTextSecondary-42 MuiTypography-alignCenter-32"
        >
          FAIRNESS / FAQ
        </span>
      </a>
      <a
        className="Component-footerLink-5"
        href="/all-prizes"
        onClick={[Function]}
        rel="noopener noreferer"
        target="_blank"
      >
        <span
          className="MuiTypography-root-9 MuiTypography-caption-19 MuiTypography-colorTextSecondary-42 MuiTypography-alignCenter-32"
        >
          ALL PRIZES
        </span>
      </a>
    </div>
  </div>
</div>
`;
